docs/BasicCommunication/OnAppRegistered/index.md

```diff
@@ -60,18 +60,52 @@ If a device is connected over Bluetooth or Wi-Fi and registers an application, S
 !!!
 
 
+###The below describes the behavior related to RSDL functionality only.   
+
+_**Changes**_   
+  1.	New appHMIType: REMOTE_CONTROL was added: all and any remote-control applications must register with this type.   
+  2.	Registration of identical applications (that is, with the same name, same VRSynonyms, same policies_appID), one from driver's device and another one from passenger's device, is allowed by RSDL.   
+  3.	Passenger's applications are not allowed to be activated to FULL level.   
+  
+_**Behavior**_
+
+I. _Application from driver's device registers_ - the vehicle must:   
+     1.	Know the device rank is driver's (see also [RC.OnDeviceRankChanged]).   
```

Comment:
Does core keep track of the device rank or does the HMI? How is the rank determined